### ISSUE TITLE
pycryptodome dependency

### ISSUE THREAD

**Issue by ravi854** [phot0n/frappe_payments#41]
Getting Following error on installation

![Screenshot_20230906_083315](https://github.com/frappe/payments/assets/85276497/a0a8d0a3-fbdb-4919-91b1-db50e4f3a30c)
![image](https://github.com/frappe/payments/assets/85276497/8f6e133c-33da-40e8-b2f7-68bbd22fb768)

**Comment by The-Debarghya** [comment]
Have you tried removing pycryptodome completely and reinstalling the compatible version?

**Comment by ravi854** [comment]
Yes, I uninstall completely and install 3.18, hence frappe also give compatibality error as Frappe require 3.10 and payment need 3.18
![image](https://github.com/frappe/payments/assets/85276497/e67f9ee4-3324-4c10-9826-966e459e7eff)

**Comment by ravi854** [comment]
Thanks 
The issue is sorted in Frappe latest update v14.49.0
